### ISSUE TITLE
Faster app opening

### ISSUE THREAD

**Issue by abidhasansojib** [Acode-Foundation/Acode#1106]
### Check for existing issues

- [ ] Completed

### Describe the feature

# Acode takes long time to open also it lags much while opening the app 
Requesting to make it more capable to load the app more faster

### If applicable, add mockups / screenshots to help present your vision of the feature

![Screenshot_2024-12-17-10-00-56-47](https://github.com/user-attachments/assets/71424bcf-880e-4776-9801-4622167309a9)

**Comment by bajrangCoder** [comment]
Yes this is most important thing, as slow startup is basically due to loading of all plugins , files/directories of sidebar at startup.

Sure we'll try to improve it

**Comment by Jobians** [comment]
@bajrangCoder if possible set size limit of files/directories to load at startup

**Comment by bajrangCoder** [comment]
> @bajrangCoder if possible set size limit of files/directories to load at startup

Actually after debugging I found that this doesn't affect much

**Comment by bajrangCoder** [comment]
We can improve speed , by changing mechanism of plugin load and loading of opened files in editor. We can also try web workers stuff.

Btw I'm experimenting with lots of stuffs to improve it.
